app.test.js

```diff
@@ -171,7 +171,7 @@ describe("/api/articles/:article_id/comments", () => {
         expect(response.body.msg).toBe("Article Not Found!");
```

Comment:
We also want to do a test when the username is not a preexisting one in the database - 404